demo/update2.json

```diff
@@ -10,6 +10,9 @@
                 "update": {
                     "attributes": {
                         "data-last-update": "update 2"
+                    },
+                    "properties": {
+                        "innerHTML": "Navigation, updated again :)"
```

Comment:
When such a Response is generated from PHP, how is it serialized ? Standard JSON serialization ?